test-app/tests/integration/components/button-test.gts

```diff
@@ -55,8 +49,6 @@ module('Integration | Component | button', function (hooks) {
       </Button>
     </template>);
 
-    assert.false(buttonPageObject.isLoading);
```

Comment:
This is the only test helper I was able to see an argument for. Is it something we need to test though? Or does the `[data-test-loading]` assertion below sufficiently cover it?

Reply:
I feel as though the assertion below is sufficient enough probably